back/boxtribute_server/graph_ql/resolvers.py

```diff
@@ -332,18 +331,32 @@ def resolve_create_qr_code(_, info, box_label_identifier=None):
 
 @mutation.field("createBox")
 @convert_kwargs_to_snake_case
-def resolve_create_box(_, info, box_creation_input):
+def resolve_create_box(*_, box_creation_input):
     authorize(permission="stock:write")
-    box_creation_input["created_by"] = g.user["id"]
-    return create_box(box_creation_input)
+    return create_box(
+        product_id=box_creation_input["product_id"],
+        location_id=box_creation_input["location_id"],
+        comment=box_creation_input["comment"],
+        items=box_creation_input["items"],
+        size_id=box_creation_input["size_id"],
+        qr_code_code=box_creation_input.get("qr_code"),
+        created_by_id=g.user["id"],
+    )
 
 
 @mutation.field("updateBox")
 @convert_kwargs_to_snake_case
-def resolve_update_box(_, info, box_update_input):
+def resolve_update_box(*_, box_update_input):
     authorize(permission="stock:write")
-    box_update_input["last_modified_by"] = g.user["id"]
-    return update_box(box_update_input)
+    return update_box(
+        label_identifier=box_update_input["label_identifier"],
+        updated_by_id=g.user["id"],
```

Comment:
In other resolvers, I used `user_id` as generic argument name, can you adjust it here?